src/components/OpenSource/WhyOpenSource.js

```diff
@@ -0,0 +1,27 @@
+import React, { Fragment } from 'react'
+import SubtitleWithBody from '../Common/SubtitleWithBody'
+import styled from 'styled-components'
+
+const StyledSubtitleWithBody = styled(SubtitleWithBody)`
+  padding-top: ${props => props.theme.spacing['1']};
+  padding-bottom: ${props => props.theme.spacing['1']};
+  margin-top: ${props => props.theme.spacing['1']};
+  margin-bottom: ${props => props.theme.spacing['2']};
+`
+
+const StyledImage = styled.img`
+  padding-top: ${props => props.theme.spacing['2']};
```

Comment:
I'm assuming that you have added `24px` padding above the image:
![image](https://user-images.githubusercontent.com/15656538/58547858-0fe83d80-8200-11e9-8a3c-57425ac2c26c.png)

so that there will be a separation between different `LogoSubtitleBody`s?

But don't forget that this means the first `LogoSubtitleBody` will also have `24px` padding above it, which we don't want because we want the top of this component to line up with the title of this section:
![image](https://user-images.githubusercontent.com/15656538/58549224-f268a300-8202-11e9-9e55-8b082ec28f5a.png)

This `24px` padding is probably best applied within the overall section component. Think [Atomic Design](http://atomicdesign.bradfrost.com/)